lib/rouge/lexers/opentypefeature.rb

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+# http://www.adobe.com/devnet/opentype/afdko/topic_feature_file_syntax.html
+
+module Rouge
+  module Lexers
+    class OpenTypeFeature < RegexLexer
+      title "OpenTypeFeature"
+      desc "OpenType feature code"
+      tag 'opentypefeature'
+      aliases 'fea', 'opentype'
+      filenames '*.fea'
+
+      def self.keywords
+        @keywords ||= %w(
+          Ascender Attach CapHeight CaretOffset CodePageRange Descender FontRevision FSType
+          GlyphClassDef HorizAxis.BaseScriptList HorizAxis.BaseTagList HorizAxis.MinMax
+          IgnoreBaseGlyphs IgnoreLigatures IgnoreMarks LigatureCaretByDev LigatureCaretByIndex
+          LigatureCaretByPos LineGap MarkAttachClass MarkAttachmentType NULL Panose RightToLeft
+          TypoAscender TypoDescender TypoLineGap UnicodeRange UseMarkFilteringSet Vendor
+          VertAdvanceY VertAxis.BaseScriptList VertAxis.BaseTagList VertAxis.MinMax VertOriginY
+          VertTypoAscender VertTypoDescender VertTypoLineGap WeightClass WidthClass XHeight
+
+          anchorDef anchor anonymous anon by contour cursive device enumerate enum
+          exclude_dflt featureNames feature from ignore include_dflt include languagesystem
+          language lookupflag lookup markClass mark nameid name parameters position pos required
+          reversesub rsub script sizemenuname substitute subtable sub table useExtension
+          valueRecordDef winAscent winDescent
+        )
+      end
+
+
+      identifier = /[a-z_][a-z0-9\/_]*/i
+
+      state :root do
+        rule %r/\s+/m, Text::Whitespace
+        rule /#.*$/, Comment
+
+        # feature <tag>
+        rule /(anonymous|anon|feature|lookup|table)((?:\s)+)/ do
+          groups Keyword, Text
+          push :featurename
+        end
+        # } <tag> ;
+        rule /(\})((?:\s|\\\s)*)/ do
+          groups Punctuation, Text
+          push :featurename
+        end
+        # solve include( ../path)
+        rule /(include)/i, Keyword, :includepath
+
+        rule /[\-\[\]\/(){},.:;=%*<>']/, Punctuation
+
+        rule /`.*?/, Str::Backtick
+        rule /\"/, Str, :dqs
+
+        # classes, start with @<nameOfClass
+        rule /@#{identifier}/, Name::Class
```

Comment:
Sorry, my previous comment was a little unclear. What I meant was that you can write this like this:

```suggestion
        rule identifier, Name::Class
```

Reply:
the re search is `@<nameOfClass>`  for example: `@lowercase` the `@` is needed + the identifier variable

Reply:
My bad—right you are!